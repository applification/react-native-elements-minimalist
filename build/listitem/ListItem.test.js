var _jsxFileName='src/listitem/ListItem.test.js';var _react=require('react');var _react2=_interopRequireDefault(_react);var _reactTestRenderer=require('react-test-renderer');var _reactTestRenderer2=_interopRequireDefault(_reactTestRenderer);var _ListItem=require('./ListItem');var _ListItem2=_interopRequireDefault(_ListItem);function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}describe('LISTITEM',function(){var title='List item title';it('Renders a list item',function(){var component=_reactTestRenderer2.default.create(_react2.default.createElement(_ListItem2.default,{title:title,__source:{fileName:_jsxFileName,lineNumber:10}})).toJSON();expect(component).toMatchSnapshot();});});