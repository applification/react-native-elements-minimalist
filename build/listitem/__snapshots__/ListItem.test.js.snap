// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LISTITEM Renders a list item 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ccc",
      "paddingBottom": 20,
      "paddingLeft": 20,
      "paddingTop": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    List item title
  </Text>
</View>
`;
